superset/utils/core.py

```diff
@@ -1737,6 +1737,8 @@ def get_time_filter_status(
             )
 
     if ExtraFiltersTimeColumnType.TIME_ORIGIN in applied_time_extras:
+        # This seems like it is to do with the druid time series, which I think we
+        # are still keeping mentions of.
```

Comment:
Can you clarify this?

Reply:
Yes! This was a misunderstanding from earlier today. I thought that we were keeping `druid_time_origin` and we are not.